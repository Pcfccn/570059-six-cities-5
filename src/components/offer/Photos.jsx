import React from "react";

const Photos = (props) => {
  const {images} = props;
  return (
    images.map((photo, i) => {
      return (
        <div className="property__image-wrapper" key={`${i}-${photo}`}>
          <img className="property__image" src={photo} alt="Photo studio"/>
        </div>);
    })
  );
};

export default Photos;
