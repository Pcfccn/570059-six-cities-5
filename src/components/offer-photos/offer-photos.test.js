import React from "react";
import renderer from "react-test-renderer";
import Photos from "./offer-photos";

it(`Should Photos render correctly`, () => {
  const tree = renderer
    .create(<Photos
      images={[`a`, `b`]}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
