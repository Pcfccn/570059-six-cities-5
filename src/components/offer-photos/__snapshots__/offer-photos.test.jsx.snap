// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Photos render correctly 1`] = `
Array [
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="a"
    />
  </div>,
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="b"
    />
  </div>,
]
`;
